vite.config.ts

```diff
@@ -25,7 +25,7 @@ export default defineConfig({
         customMedia: true,
       },
       targets: {
-        chrome: 131,
+        chrome: 131 << 16,
```

Comment:
What are we solving by setting the target to that?

Reply:
https://github.com/parcel-bundler/lightningcss/issues/773#issuecomment-2270330817

Reply:
Huh, wow 🤯 👍